api/test/services/gitland/commands/pull_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+module Gitland
+  module Commands
+    class PullTest < ActiveSupport::TestCase
+      def setup
+        @repository = repositories(:rails)
+      end
+
+      test "#execute pulls the repository" do
+        repo = Git.open(Rails.root.join("storage", "gitland", "repositories", "1.git").to_s)
+        repo.expects(:pull)
+
+        Git.stubs(:open).returns(repo)
+
+        Gitland::Commands::Pull.new(@repository).execute
```

Comment:
Yeah this path does not exists in the test environment. This won't work. 

We can probably just assert that `Git.open` is called with the right arguments, and assert that the `#pull` method is called. 

There's examples of stubbing and expectations in:
- https://github.com/visevol/GithubVisualisation/blob/main/api/test/services/gitland/commands/log_test.rb
- https://github.com/visevol/GithubVisualisation/blob/main/api/test/services/gitland/commands/diff_test.rb

Reply:
fixed